### PR TITLE
Docs: Add links to project settings

### PR DESCRIPTION
**Pull request by tetrapod00**
Changes a few mentions of project settings that were styled like **Application > Run > Main Scene** rather than `ProjectSettings.application/run/main_scene`.

### CODE REVIEW

File: doc/classes/PhysicalBone3D.xml [hunk 1]
```diff
@@ -48,8 +48,7 @@
 	</methods>
 	<members>
 		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" default="0.0">
-			Damps the body's rotation. By default, the body will use the [b]Default Angular Damp[/b] in [b]Project &gt; Project Settings &gt; Physics &gt; 3d[/b] or any value override set by an [Area3D] the body is in. Depending on [member angular_damp_mode], you can set [member angular_damp] to be added to or to replace the body's damping value.
-			See [member ProjectSettings.physics/3d/default_angular_damp] for more details about damping.
```
Comment (tetrapod00) [inline doc/classes/PhysicalBone3D.xml:52]: Note that in several cases, I removed the second paragraph which *did* link to the project setting, assuming that one link in the first paragraph would be sufficient.

I'm not sure if that's the right call. Do we need these second paragraphs explicitly calling attention to the [additional documentation](https://docs.godotengine.org/en/latest/classes/class_projectsettings.html#class-projectsettings-property-physics-2d-default-angular-damp) that exists in the project settings?

Comment (Mickeon) [reply doc/classes/PhysicalBone3D.xml:52]: > Do we need these second paragraphs explicitly calling attention to the additional documentation that exists in the project settings?

My opinion, no. A lot of the "for more details" are redundantly obviously.

However, there's a lot of explanation being done in the Project Settings's HUGE description. Hence the reminder makes some sort of sense. It's still kind of bad that there's little to no summary on what damping is in this description, but this was probably done to focus all documentation in one spot.

Comment (tetrapod00) [reply doc/classes/PhysicalBone3D.xml:52]: Should I err on the side of leaving in the redundant second paragraph, then?

Comment (Mickeon) [reply doc/classes/PhysicalBone3D.xml:52]: For the moment, yeah. Especially if you're not completely overhauling the descriptions (which I wouldn't recommend spending time on, as this PR is a pretty good thing to do as is).

Comment (tetrapod00) [reply doc/classes/PhysicalBone3D.xml:52]: Done :)

**Review by Mickeon — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by Repiteo** [comment]
Needs to be rebased before this can be merged

**Comment by tetrapod00** [comment]
Rebased.

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**